runtime-sdk/src/modules/consensus_accounts/mod.rs

```diff
@@ -569,6 +569,14 @@ impl<Consensus: modules::consensus::API> Module<Consensus> {
         state::get_delegation(args.from, args.to)
     }
 
+    #[handler(call = "consensus.Delegation", internal)]
+    fn internal_delegation<C: Context>(
+        _ctx: &C,
+        args: types::DelegationQuery,
+    ) -> Result<types::DelegationInfo, Error> {
```

Comment:
This is missing gas accounting. See other internal calls as it should not be free to use :)

Reply:
Oops, fixed :)